samples/rest/BingWebSearchV7.py

```diff
@@ -7,33 +7,41 @@
 import os
 from pprint import pprint
 import requests
+from dotenv import load_dotenv
+
+# Load the environment variables from .env file
+load_dotenv()
 
 '''
 This sample makes a call to the Bing Web Search API with a query and returns relevant web search.
 Documentation: https://docs.microsoft.com/en-us/bing/search-apis/bing-web-search/overview
 '''
+AUTH_HEADER=os.environ.get('BING_SEARCH_V7_AUTHORIZATION_HEADER', 'Ocp-Apim-Subscription-Key')
+API_KEY='BING_SEARCH_V7_WEB_SEARCH_SUBSCRIPTION_KEY'
```

Comment:
Also OK but I think the variable names could be clearer.

For API_KEY it would be nice to indicate it's the name of an environment variable, and it's for subscription_key. Something like SUBSCRIPTION_KEY_ENV_VAR_NAME but that's pretty verbose, and elsewhere we're hard-coding environment variable names. -- maybe it'd be best to keep this hard-coded as well?

Same thing for AUTH_HEADER if we keep it an environment variable. Could we call it AUTH_HEADER_NAME?

Reply:
Done: aaee62ec4862618af122513ffc88f05ec2669a79